packages/workspace/src/generators/convert-to-monorepo/convert-to-monorepo.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  convertNxGenerator,
+  getProjects,
+  joinPathFragments,
+  ProjectConfiguration,
+  readNxJson,
+  Tree,
+  updateNxJson,
+} from '@nx/devkit';
+import { moveGenerator } from '../move/move';
+import { Schema } from './schema';
+
+export async function monorepoGenerator(tree: Tree, options: Schema) {
+  const projects = getProjects(tree);
+  mayebExtractTsConfigBase(tree);
+  await maybeExtractJestConfigBase(tree);
+
+  const nxJson = readNxJson(tree);
+  nxJson.workspaceLayout = {
+    // normalize paths without trailing slash
+    appsDir: joinPathFragments(options.appsDir),
+    libsDir: joinPathFragments(options.libsDir),
+  };
+
+  updateNxJson(tree, nxJson);
+
+  for (const [_name, project] of projects) {
+    maybeExtractEslintConfigIfRootProject(tree, project);
+    await moveGenerator(tree, {
+      projectName: project.name,
+      destination: project.name,
```

Comment:
Pass `destinationRelativeToRoot` so that the destination is taken as is.

This turns into `join(options.appsDir | options.libsDir, project.name)`